src/abi/program.rs

```diff
@@ -22,6 +22,7 @@ pub struct ProgramABI {
     pub messages_types: Option<Vec<MessageType>>,
     pub configurables: Option<Vec<Configurable>>,
     pub error_codes: Option<BTreeMap<u64, ErrorDetails>>,
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
I remember when adding `error_codes`, I didn't mark them as `skip_serializing` only because all other `Option`s of collections are not marked as such. The unfortunate consequence was, as explained in this PR description, adapting all the ABI oracles in test. 

What do you think about marking all of them as `skip_serializing`? It would be consistent. With this change, the `panicking_calls` stick out without a clear reason.